From d1025eaaadb9cdd86f20f1334a58feb2a23a2f6a Mon Sep 17 00:00:00 2001
From: Cole Mickens <cole.mickens@gmail.com>
Date: Fri, 1 Jan 2016 11:29:38 -0800
Subject: [PATCH] add 'decorated' config option to disable csd

---
 termite.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/termite.cc b/termite.cc
index 55b79e4..7cb973a 100644
--- a/termite.cc
+++ b/termite.cc
@@ -1339,6 +1339,8 @@ static void set_config(GtkWindow *window, VteTerminal *vte, config_info *info,
     info->modify_other_keys = cfg_bool("modify_other_keys", FALSE);
     info->fullscreen = cfg_bool("fullscreen", TRUE);
 
+    gtk_window_set_decorated(window, cfg_bool("decorated", TRUE));
+
     g_free(info->browser);
     info->browser = nullptr;
 
-- 
2.9.2

